frontend/templates/_footer.html

```diff
@@ -38,10 +38,10 @@
       <div class="col-sm-4">
         <ul class="list-unstyled">
           <li><strong>Information</strong></li>
-          <li><a href="http://legacy.pmg.org.za/rules-parliament-guide-procedure">Rules of Parliament</a></li>
+          <li><a href="/page/rules-parliament-guide-procedure">Rules of Parliament</a></li>
           <li><a href="/page/code-of-ethical-conduct">Code of Conduct / MP Disclosure of Interests</a></li>
           <li><a href="/page/political-party-representation">Political Party Representation</a></li>
-          <li><a href="http://legacy.pmg.org.za/legislative-process-government-links">Legislative Process &amp; Government</a></li>
+          <li><a href="/page/legislative-process-government-links">Legislative Process &amp; Government</a></li>
```

Comment:
The page this goes to has a broken link, the very first one two `http://www.pmg.org.za/node/17034`